lib/UserInterface/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ endif()
 
 add_cling_library(clingUserInterface
   UserInterface.cpp
+  ClingCodeCompleteConsumer.cpp
```

Comment:
I think the patch does not contain ClingCodeCompleteConsumer.cpp? Did you git add it?